src/NuGet.Core/NuGet.Packaging/Signing/Timestamp/Rfc3161TimestampVerificationUtility.cs

```diff
@@ -19,17 +15,15 @@ internal static class Rfc3161TimestampVerificationUtility
     {
         private const double _millisecondsPerMicrosecond = 0.001;
 
-#if IS_DESKTOP
-
         internal static bool ValidateSignerCertificateAgainstTimestamp(
             X509Certificate2 signerCertificate,
             Timestamp timestamp)
         {
             DateTimeOffset signerCertExpiry = DateTime.SpecifyKind(signerCertificate.NotAfter, DateTimeKind.Local);
             DateTimeOffset signerCertBegin = DateTime.SpecifyKind(signerCertificate.NotBefore, DateTimeKind.Local);
 
-            return timestamp.UpperLimit < signerCertExpiry &&
-                timestamp.LowerLimit > signerCertBegin;
+            return timestamp.UpperLimit <= signerCertExpiry &&
```

Comment:
This seemed like an error in our code. If this is incorrect we can revert this change and make sure the tests take into account so they don't fail.